src/tasks/cmd/param_optimization.rs

```diff
@@ -155,13 +156,23 @@ fn add_scalables(
         },
 
         (
-            &cfg::Scalable::UniformLayerSep { .. },
-            &ScalableCoords::UnknownLayers { .. },
-        ) | (
-            &cfg::Scalable::LayerSeps { .. },
+            &cfg::Scalable::UniformLayerSep { .. } | &cfg::Scalable::LayerSeps { .. },
             &ScalableCoords::UnknownLayers { .. },
         ) => bail!("cannot scale layer separations when layers have not been determined"),
 
+        (
+            &cfg::Scalable::UniformLayerSep { ref mask, .. } | &cfg::Scalable::LayerSeps { ref mask, .. },
```

Comment:
hmmm I wonder why these changes are showing up here, did I not merge them into master?